### PR TITLE
chore(http): Drop lodash dependency

### PR DESCRIPTION
**Pull request by gjou-tlnd**
**What is the problem this PR is trying to solve?**

Lodash can't be used in service worker due to current ui-scripts config: lodash is expected as an external but it's not there
image

**What is the chosen solution to this problem?**

Config should be fixed but we can also remove lodash as it's not really needed

**Please check if the PR fulfills these requirements**

* [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jsomsanith-tlnd — APPROVED** [review]

**Review by frassinier — APPROVED** [review]
Ok if you don't introduce breaking changes

**Comment by gjou-tlnd** [comment]
> Ok if you don't introduce breaking changes

Test suite is green, everything should be ok 😶

File: packages/http/src/csrfHandling.js [hunk 1]
```diff
@@ -57,12 +53,12 @@ function parseCookie(cookie) {
  * @param {Object.CSRFTokenCookieKey} CSRFTokenCookieKey - default `csrfToken`
  * @param {Map.<string, string>} cookieValues
  */
-const findCSRFToken = curry(({ CSRFTokenCookieKey = 'csrfToken' }, cookieValues) => {
+const findCSRFToken = ({ CSRFTokenCookieKey = 'csrfToken' }) => cookieValues => {
```
Comment (jmfrancois) [inline packages/http/src/csrfHandling.js:56]: curry do not mean that.
You may break users of http here

**Review by jmfrancois — APPROVED** [review]
good but with a comment !

File: packages/http/src/csrfHandling.js (hunk 1 above)
Comment (gjou-tlnd) [reply packages/http/src/csrfHandling.js:56]: do you have an example of what could go wrong? This function is not exported and I made sur the only usage is OK

**Review by romainseb — APPROVED** [review]

**Pull request merged**